Assets/Scripts/UI/HealthEffectController.cs

```diff
@@ -0,0 +1,134 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+public class HealthEffectController : MonoBehaviour
+{
+   
+   public Volume volume;
+
+   private Vignette vignette;
+   public bool vignetteToggle;
+
+   private FilmGrain filmgrain;
+   public bool filmgrainToggle;
+
+   private PaniniProjection paniniprojection;
+   public bool distanceToggle;
+   public bool cropToFitToggle;
+
+   private LensDistortion lensdistortion;
+   public bool lensDistortionToggle;
+
+   private Bloom bloom;
+   public bool bloomToggle;
+
+   public Killable killable;
+
+   void Start()
+   {
+
+    if(volume.profile.TryGet<Vignette>(out vignette))
```

Comment:
It looks like you arent setting any of the components you are getting here to the variables you declare above